mikeio/pfs.py

```diff
@@ -191,7 +191,7 @@ def catch_str_exceptions(self, v):
                     v = f"'{v}'"
 
         if isinstance(v, datetime):
-            #v = v.strftime("%Y, %#m, %#d, %#H, %M, %S") # pfs-datetime output
+            # v = v.strftime("%Y, %#m, %#d, %#H, %M, %S") # pfs-datetime output
             v = v.strftime("%Y, %-m, %-d, %-H, %-M, %-S")  # no zero padding
```

Comment:
this notation works only on unix-based systems

Reply:
I see, too bad.